### DISCUSSION TITLE
堆叠模式下不能按照Zindex层级渲染

### DISCUSSION THREAD

**Discussion by paiap** [didi/LogicFlow#1189] [Q&A]
我有两个分组的节点，一个是领域，一个是应用，他们好像默认在基本组件几点的下面，但是不知道为什么，我想让领域这个分组节点一直在领域上方，但是一直不行，使用toFront，等调整Zindex的方法也不生效

**Comment by towersxu** [comment]
参考源码中默认设置了zIndex。https://github.com/didi/LogicFlow/blob/2edd2eaf00544d874f12a4d5633cd97386952040/packages/extension/src/materials/group/GroupNode.ts#LL52C9-L52C9
